int64 id
---
int64 new_id